destination/destination.go

```diff
@@ -12,12 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:generate paramgen -output=paramgen_dest.go Config
```

Comment:
I'd move this declaration to `config.go` to keep it close to the `Config` struct (same in source).